pkg/cmd/triggerbinding/list.go

```diff
@@ -61,9 +66,18 @@ or
 				return err
 			}
 
-			tbs, err := list(cs.Triggers, p.Namespace())
+			namespace := p.Namespace()
+			if opts.AllNamespaces {
+				namespace = ""
+			}
+
+			tbs, err := list(cs.Triggers, namespace)
 			if err != nil {
-				return fmt.Errorf("failed to list TriggerBindings from %s namespace: %v", p.Namespace(), err)
+				ns := namespace
+				if opts.AllNamespaces {
+					ns = "all"
+				}
+				return fmt.Errorf("failed to list TriggerBindings from %s namespace: %v", ns, err)
```

Comment:
```suggestion
				return fmt.Errorf("failed to list TriggerBindings from %s namespaces: %v", ns, err)
```

Reply:
Should we do if-else here? Suppose if now triggerBinding from a single namespace is failed to list then an error will look like `failed to list TriggerBindings from test-123 namespaces: `. It looks grammatically wrong while reading. WDYS?

Reply:
Yes, I see your point. I like the if-else idea.

Reply:
I've updated PTAL.